package.json

```diff
@@ -1,43 +1,46 @@
 {
-  "name": "tutorial",
+  "name": "recipe-app",
   "version": "1.0.0",
   "scripts": {
-    "devStart": "nodemon server.js",
+    "devStart": "nodemon --config nodemon.json ./src/server.ts",
```

Comment:
You should either have nodemon.json in your project folder (not globally), or use nodemonConfig in package.json, not both.